packages/components/text/src/text.styles.ts

```diff
@@ -43,7 +43,7 @@ const getTone = (tone: string) => {
     case 'secondary':
       return `color: ${designTokens.colorNeutral40};`;
     case 'positive':
-      return `color: ${designTokens.colorPrimary25};`;
+      return `color: ${designTokens.fontColorForTextAsPositive};`;
```

Comment:
That's an additional change not listed in the Figma file

Reply:
Got a late request from Filip. Hope Filip won't mind me sharing the DM: 
```
In the text components we have the tone prop in `Caption` `Detail` `Body` `Subheadline`. In those components we would need to change the `tone: positive` to use `color-success-40`
```
cc @FilPob

Reply:
Thanks for sharing.

I think it would be beneficial to, aside of the DM, to also update the Figma file so we can have the requirements updated in case we need to check this again in the future so we know we implemented what we've been asked for.

cc/ @FilPob